stripper/ze_lotr_minas_tirith_p5.cfg

```diff
@@ -1,3 +1,211 @@
+	"_quadratic_attn" "1"
+	"_lightHDR" "-1 -1 -1 1"
+	"_light" "0 50 250 1500"
+	"_inner_cone" "0"
+	"_cone" "0"
+	"classname" "light_dynamic"
+}
+
+add:
+{
+	"classname" "trigger_teleport"
+	"targetname" "A7helper_tp_ct"
+	"target" "spawn_destination_ct"
+	"spawnflags" "1"
+	"origin" "12256 930 10040"
+	"model" "*387"
+	"UseLandmarkAngles" "1"
+	"StartDisabled" "0"
+}
+
+add:
+{
+	"classname" "info_teleport_destination"
+	"targetname" "spawn_destination_ct"
+	"origin" "9871 3000 10100"
+	"angles" "0 270 0"
+}
+
+add:
+{
+	"origin" "6974 1056 10088"
+	"targetname" "A7helper_t"
+	"style" "0"
+	"spotlight_radius" "1600"
+	"spawnflags" "0"
+	"pitch" "-10"
+	"distance" "0"
+	"brightness" "2"
+	"angles" "0 0 0"
+	"_quadratic_attn" "1"
+	"_lightHDR" "-1 -1 -1 1"
+	"_light" "255 0 0 1500"
+	"_inner_cone" "0"
+	"_cone" "0"
+	"classname" "light_dynamic"
+}
+
+add:
+{
+	"classname" "trigger_teleport"
+	"targetname" "A7helper_tp_t"
+	"target" "spawn_destination_t"
+	"spawnflags" "1"
+	"origin" "6974 1056 10088"
+	"model" "*387"
+	"UseLandmarkAngles" "1"
+	"StartDisabled" "1"
+}
+
+add:
+{
+	"classname" "info_teleport_destination"
+	"targetname" "spawn_destination_t"
+	"origin" "11134 497 10147"
+	"angles" "0 130 0"
+}
+
+add:
+{
+	"origin" "12256 930 10080"
+	"targetname" "A7helper_ct_sp"
+	"spawnflags" "1"
+	"scale" "0.1"
+	"rendermode" "5"
+	"renderfx" "0"
+	"rendercolor" "0 50 250"
+	"renderamt" "255"
+	"parentname" "item_tnt_1"
+	"model" "pmaterials/640_train.vmt"
+	"mindxlevel" "0"
+	"maxdxlevel" "0"
+	"HDRColorScale" "1.0"
+	"GlowProxySize" "2.0"
+	"framerate" "0"
+	"fadescale" "1"
+	"fademindist" "-1"
+	"disablereceiveshadows" "0"
+	"_minlight" "0.0"
+	"classname" "env_sprite"
+}
+
+add:
+{
+	"origin" "6974 1056 10088"
+	"targetname" "A7helper_t_sp"
+	"spawnflags" "0"
+	"scale" "0.1"
+	"rendermode" "5"
+	"renderfx" "0"
+	"rendercolor" "255 0 0"
+	"renderamt" "255"
+	"parentname" "item_tnt_1"
+	"model" "pmaterials/640_train.vmt"
+	"mindxlevel" "0"
+	"maxdxlevel" "0"
+	"HDRColorScale" "1.0"
+	"GlowProxySize" "2.0"
+	"framerate" "0"
+	"fadescale" "1"
+	"fademindist" "-1"
+	"disablereceiveshadows" "0"
+	"_minlight" "0.0"
+	"classname" "env_sprite"
+}
+
+;change spawn teleporters + reformat message
+modify:
+{
+	match:
+	{
+		"classname" "logic_relay"
+		"targetname" "logica_start"
+	}
+	delete:
+	{
+		"OnTrigger" "consolaCommandsay ** THE ROUND HAS STARTED ** TAKE ONE OF THE PORTALS ** MASSIVE TELEPORTATION IN 20 SECONDS **0-1"
+	}
+	insert:
+	{
+		"OnTrigger" "A7helper_tp_ctkill0-1"
+		"OnTrigger" "A7helper_ctdistance00-1"
+		"OnTrigger" "A7helper_ct_spkill0-1"
+		"OnTrigger" "A7helper_tp_tEnable0-1"
+		"OnTrigger" "A7helper_tdistance10000-1"
+		"OnTrigger" "A7helper_t_spShowSprite0-1"
+		"OnTrigger" "consolaCommandsay ** THE ROUND HAS STARTED **0-1"
+		"OnTrigger" "consolaCommandsay ** TAKE ONE OF THE PORTALS **0.1-1"
+		"OnTrigger" "consolaCommandsay ** MASSIVE TELEPORTATION IN 20 SECONDS **0.2-1"
+	}
+}
```

Comment:
I don't think this is at all necessary, I verified all spawn groupings are very similar distances from the middle, all with similar sizes.

> the need for constantly killing yourself to get a better spawn

This is a choice.

Reply:
The main goal of this change wasn't to make it so one side can make it closer to middle first. It was so people who get "bad spawn" can still teleport to the other side in time to go into the item shop. 

Even if players can kill themselves for better spawns, there's no guarantees that you'll spawn closer to the shop.

Reply:
Then this is no different than the mako situation, we're not going to block off the back trains just because people want a front spawn for items.

Reply:
Alright. I'll remove this portion then.